<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>power_system_data</class>
 <widget class="QWidget" name="power_system_data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>636</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000000</width>
    <height>1000000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QWidget" name="power_system_data_widget" native="true">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="QFrame" name="frame_3">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(158, 158, 158);
border-radius: 25px;
background-color: rgb(193, 129, 0);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="3">
          <widget class="QFrame" name="power_system_info_frame">
           <property name="styleSheet">
            <string notr="true">
border-color: rgb(129, 194, 65);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>7</number>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="bus_label">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Buses (Zones): --</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="line_label">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Branches: --</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="gen_label">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255)</string>
              </property>
              <property name="text">
               <string>Generators:  --</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="sys_label">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>System Name: --</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QFrame" name="frame_5">
           <property name="maximumSize">
            <size>
             <width>10000000</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="3" column="3">
             <widget class="QPushButton" name="open_file_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(129, 194, 65);
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Open</string>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>System Name</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Data Folder</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1" colspan="3">
             <widget class="QComboBox" name="file_combo_box">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
              </property>
              <property name="currentText">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="3">
             <widget class="QLineEdit" name="system_name_input">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
              </property>
              <property name="text">
               <string>Default System Name</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QPushButton" name="file_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(44, 81, 187);
color: rgb(255, 255, 255);
background-color: rgb(40, 84, 113);</string>
              </property>
              <property name="text">
               <string>Browse</string>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QToolButton" name="power_system_help_button">
              <property name="styleSheet">
               <string notr="true">QToolButton{
image: url(:/icon/images/icons/info_help1.png);
	background-color: rgb(226, 226, 226);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="Line" name="divider_line">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(45, 82, 190);</string>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="power_system_data_frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(45, 82, 190);
border-radius: 25px;
background-color: rgb(93, 186, 0);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1" rowspan="2">
          <widget class="MatplotlibWidget" name="network_map_widget">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3" rowspan="2">
          <widget class="MatplotlibWidget" name="generation_mix">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_bottom">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>10000000</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame_6">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="previous_button">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(208, 69, 0);
font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Previous</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/logos/images/logo/power_system_icon.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="next_button">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(129, 194, 65);
font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Next</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatplotlibWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">matplotlibwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
